glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -7,6 +7,32 @@ package mozilla.telemetry.glean.config
 import mozilla.components.concept.fetch.Client
 import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient
 import mozilla.telemetry.glean.BuildConfig
+import mozilla.telemetry.glean.rust.toByte
+
+import com.sun.jna.Structure
+import com.sun.jna.ptr.LongByReference
+
+@Structure.FieldOrder("dataDir", "packageName", "uploadEnabled", "maxEvents")
+class FfiConfiguration(
+    dataDir: String,
+    packageName: String,
+    uploadEnabled: Boolean,
+    maxEvents: Long? = null
+) : Structure() {
+    @JvmField
+    public var dataDir: String = dataDir
+    @JvmField
+    public var packageName: String = packageName
+    @JvmField
+    public var uploadEnabled: Byte = uploadEnabled.toByte()
+    @JvmField
+    public var maxEvents: LongByReference = if (maxEvents == null) LongByReference() else LongByReference(maxEvents!!)
+
+    init {
+        // String encoding UTF8로 강제
+        this.stringEncoding = "UTF-8"
```

Comment:
Why is this required?

Reply:
I ... copied it over and I'm not sure if it is _required_. Let me pull up ~~the (non-existing) documentation~~ implementation code.